plugins/modules/aci_infra_port_config.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Sabari Jaganathan <sajagana@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_infra_port_config
+short_description: Manage the Port Configuration of the Fabric Access Policies - Interface Configuration (infra:PortConfig)
+description:
+- Manage the Port Configuration of the Fabric Access Policies - Interface Configuration on Cisco ACI fabrics.
+options:
+  assoc_grp:
+    description:
+    - The Associated Group DN of the Access Port Policy Group.
+    type: str
+    aliases: [ access_port_policy_group ]
+  brkout_map:
+    description:
+    - The Breakout Map of the interface and assoc_grp should be empty while configuring the Breakout Map.
+    type: str
+    choices: [ 100g-4x, 10g-4x, 25g-4x ]
+    aliases: [ breakout_map ]
+  card:
+    description:
+    - The slot number of the Network Interface Card(NIC) and the Card ID must be between 1 to 64.
+    type: int
+  description:
+    description:
+    - Description of the Interface Port Configuration object.
+    type: str
+    aliases: [ descr ]
+  node:
+    description:
+    - The ID of the Node and the value must be between 101 to 4000.
+    type: int
+    aliases: [ node_id ]
+  pc_member:
+    description:
+    - The name of the Port Channel Member.
+    type: str
+    aliases: [ port_channel_member ]
+  port_type:
+    description:
+    - The type of the interface port can be either access or fabric and the default port type is access.
+    type: str
+    default: access
+    choices: [ access, fabric ]
+  port_id:
+    description:
+    - The Port ID of the Network Interface Card(NIC) and the Port ID must be between 1 to 128.
+    type: int
+    aliases: [ port_channel_member ]
```

Comment:
port_channel_member is already aliases of pc_member. Do we need both pc_member and port_id?